scripts/competitions/svcomp/esbmc-wrapper.py

```diff
@@ -206,7 +206,7 @@ def get_result_string(the_result):
 # ESBMC default commands: this is the same for every submission
 esbmc_dargs = "--no-div-by-zero-check --force-malloc-success --state-hashing "
 esbmc_dargs += "--no-align-check --k-step 2 --floatbv --unlimited-k-steps "
-esbmc_dargs += "--no-por --context-bound-step 5 --max-context-bound 15 "
+esbmc_dargs += "--no-por "
```

Comment:
shouldn't `--no-por` be only used on when running the `concurrency` mode?

Reply:
Yes. It should be removed, @rafaelsamenezes.